tests/foreman/cli/test_remoteexecution.py

```diff
@@ -367,3 +424,137 @@ def test_positive_run_scheduled_job_template(self):
                     )
                 )
             )
+
+    @tier3
+    def test_positive_run_default_job_template_multiple_hosts(self):
+        """Run default job template against multiple hosts
+
+        :id: 415c0156-be77-4676-918b-c0d4be810b0e
+
+        :expectedresults: Verify the job was successfully ran against all hosts
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as client2:
```

Comment:
can you add a `tearDown` to the tests involving VMs to perform a cleanup after the job is finished? (`client2.destroy()`)